index.js

```diff
@@ -51,16 +51,24 @@ const body = {
   parameters: parameters,
 };
 
+const tag = getTag();
+const branch = getBranch();
+
 if (tag) {
-  Object.assign(parameters, { GHA_Tag: tag() });
```

Comment:
`GHA_Tag` and `GHA_Branch` were not documented in the readme and if they were set would have caused the circle build to fail as they wouldn't have been defined on the circle config. It doesn't make sense to pass them as parameters as they can be determined from within any job inside of circleci so I just removed them

Reply:
You are correct! I was trying to figure out why this code would be here (my fault obviously), and I _think_ I found the broken train of thought. I think the original intent may have been to append "branch" and "tag" to the body. 

You are correct we would not need either of those parameters, which led me to question why I put them there. I think the intent was the ensure the pipeline was _triggered_ on a branch or tag, and it appears that is likely not working now.

https://circleci.com/docs/api/v2/#operation/triggerPipeline

Payload sample: 

```json
{
  "branch": "feature/design-new-api",
  "tag": "v3.1.4159",
  "parameters": {
    "deploy_prod": true
  }
}
```

Rather than remove them I think we might want to make a few adjustments to ensure the branch/tag key are added to the body so that the pipeline is triggered at the right reference

Reply:
For sure! That's what we're now doing here (and was the main source of the bug):

https://github.com/CircleCI-Public/trigger-circleci-pipeline-action/blob/d0dabb5cf1f280ecdd1398f37fac2e480a534a04/index.js#L57-L61

Reply:
Based on how we check the `ref` field it looks like it's not possible to have both a tag AND a branch so I think it's correct to only pass one or the other

https://github.com/CircleCI-Public/trigger-circleci-pipeline-action/blob/d0dabb5cf1f280ecdd1398f37fac2e480a534a04/index.js#L22

https://github.com/CircleCI-Public/trigger-circleci-pipeline-action/blob/d0dabb5cf1f280ecdd1398f37fac2e480a534a04/index.js#L28